PWGEM/PhotonMeson/Tasks/gammaConversions.cxx

```diff
@@ -494,18 +602,73 @@ struct GammaConversions {
     }
   }
 
+bool RVsZPassLineCut(float v0radius, float z)
+{
+  if(TMath::Abs(z) < LineCutZ0 + v0radius * LineCutZRSlope)
+  {
+    return true;
+  }
+  return false;
+}
+
+  //this is really ugly but it works. Would be easier if .end() worked..
+  template <typename TTRACKS>
+  void fillhpeDivpGammaHistograms(mapStringHistPtr& theContainer, TTRACKS const& theTwoV0Daughters)
+  {
+    float p_electron[2];
+    Int_t counter = 0;
+
+    for (auto& lTrack : theTwoV0Daughters)
+    {
+      p_electron[counter] = lTrack.p();
+      counter++;
+    }
+
+    fillTH1(theContainer, "hpeDivpGamma", p_electron[0]/(p_electron[0]+p_electron[1]));
```

Comment:
I think these quanttities (p_p/p_tot and p_n/p_tot) should be added as dynamic columns to the V0datas table in StrangenessTables.h  . Then you have aceess to the quantities directly from theV0 and you can put the fill calls into fillV0Histograms

Reply:
This was wrong anyway. I modefied it since it needed the V0 and the Track momentum. Dynamic columns thus not possible, because it uses data from 2 different tables.